easybuild/easyblocks/t/tensorflow.py

```diff
@@ -208,8 +208,8 @@ def configure_step(self):
             'TF_NEED_OPENCL': ('0', '1')[bool(opencl_root)],
             'TF_NEED_OPENCL_SYCL': '0',
             'TF_NEED_S3': '0',  # Amazon S3 File System
-            'TF_NEED_VERBS': '0',
             'TF_NEED_TENSORRT': ('0', '1')[bool(tensorrt_root)],
```

Comment:
@akesandgren I really mean `0` here, since it's revisited correctly later:

```suggestion
             'TF_NEED_TENSORRT': '0',

Reply:
It isn't changed below anymore.

Reply:
But you're right. It shouldn't be set to 1 unless cuda is set. I'll change it back again.